vmdb/app/models/ems_refresh/parsers/openshift.rb

```diff
@@ -34,7 +34,7 @@ def parse_route(route)
             :host_name    => route.host,
             :labels       => parse_labels(route),
             # TODO: this part needs to be modified to service_id instead
-            :service_name => route.serviceName,
+            :container_service_id => route.container_service_id,
```

Comment:
and add:

    new_result[:container_service] = @data_index.fetch_path(:container_services, :by_namespace_and_name, new_result[:namespace], route.serviceName)